### DISCUSSION TITLE
Which storybook version works under node version 12

### DISCUSSION THREAD

**Discussion by Lord-Leonard** [storybookjs/storybook#23922] [Help]
Hello there, 

I am currently working on a Projekt stuck on node version 12. 
We will start introducting web components and I was wondering if any storybook version is running under node v12. 

We will use standard js/html webcomponents so we are not dependend on any fancy feature. It would just be nice to have the basic storybook functionality. 

Thanks in advance
Leo

**Comment by Lord-Leonard** [comment]
ping?

**Reply by njdxd** [reply]
test reply

**Reply by njdxd** [reply]
cont

**Comment by johnhunter** [comment]
I have a feeling v6 will work with Node@12 but bear in mind it is old and will not receive patches and updates. See https://storybook.js.org/docs/6.5/html/get-started/install
